tests/unit/amo/components/TestContribute.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { oneLine } from 'common-tags';
+
+import Contribute from 'amo/components/Contribute';
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_LANG,
+  ADDON_TYPE_THEME,
+} from 'core/constants';
+import Button from 'ui/components/Button';
+import Card from 'ui/components/Card';
+import { fakeI18n } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const render = (props = {}) => {
+    const allProps = {
+      contributionsURL: 'https://paypal.me/babar',
+      addonType: ADDON_TYPE_EXTENSION,
+      numberOfAuthors: 1,
+      i18n: fakeI18n(),
+      ...props,
+    };
+
+    return shallow(<Contribute {...allProps} />, {
+      context: { i18n: allProps.i18n },
+    }).shallow();
+  };
+
+  it('renders a Card', () => {
+    const root = render();
+    expect(root.find(Card)).toHaveLength(1);
+  });
+
+  it('renders a Button with a contributions URL', () => {
+    const root = render();
+    expect(root.find(Button)).toHaveLength(1);
+    expect(root.find(Button)).toHaveProp('href', 'https://paypal.me/babar');
+    expect(root.find(Button).children()).toContain('Contribute now');
+  });
+
+  it('displays content for an extension developer', () => {
+    const root = render();
+    expect(root.find(Card)).toHaveProp('header', 'Support this developer');
+    expect(root.find('.Contribute-content'))
+      .toHaveText(oneLine`The developer of this extension asks that you help
+        support its continued development by making a small contribution.`);
+  });
+
+  it('displays content for multiple extension developers', () => {
+    const root = render({ numberOfAuthors: 3 });
+    expect(root.find(Card)).toHaveProp('header', 'Support these developers');
+    expect(root.find('.Contribute-content'))
+      .toHaveText(oneLine`The developers of this extension ask that you help
```

Comment:
Since these tests only check that the alternate translation is *used*, you don't have to include the full string in the assertion. You could just check a subset: `.toMatch(/The developers of this extension/)`. That should cut down on having to fix tests when making minor changes to the copy.